ambry-store/src/main/java/com/github/ambry/store/BlobStoreCompactor.java

```diff
@@ -85,6 +86,7 @@ public boolean accept(File dir, String name) {
   private final AccountService accountService;
   private volatile boolean isActive = false;
   private PersistentIndex srcIndex;
+  private final Set<Pair<Short,Short>> notActiveContainers;
```

Comment:
minor and optional: I feel like `deprecatedContainers` sounds a better name.

Reply:
`deprecatedContainers ` also looks good to me.